pkgs/sketch_pad/lib/editor/editor.dart

```diff
@@ -238,10 +259,14 @@ class _EditorWidgetState extends State<EditorWidget> implements EditorService {
         widget.appModel.editorStatus.showToast('No quick fixes available.');
       }
 
-      return HintResults.fromHints([
-        ...response.fixes.map((change) => change.toHintResult()),
-        ...response.assists.map((change) => change.toHintResult()),
-      ], doc.posFromIndex(sourceOffset), doc.posFromIndex(0));
+      return HintResults(
+        list: [
+          ...response.fixes.map((change) => change.toHintResult(codeMirror!)),
+          ...response.assists.map((change) => change.toHintResult(codeMirror!)),
```

Comment:
Since this is already pulled out into a separate variable earlier:

```suggestion
          ...response.fixes.map((change) => change.toHintResult(editor)),
          ...response.assists.map((change) => change.toHintResult(editor)),
```